### PR TITLE
simplify workflow section.

### PR DESCRIPTION
**Pull request by matt-maguire**
Now that scene-referred workflow is the darktable default, this PR proposes to simplify the workflow overview section, and align it to the new modules available in DT 3.6/3.7. Detailed/technical non-recommended module information has been pushed to an appendix.

### CODE REVIEW

File: content/overview/workflow/edit-overview.md [hunk 1]
```diff
@@ -10,33 +10,39 @@ This section will guide you through the basics of developing an image in the [da
...
 
-On the left panel of the darkroom view is the [history stack](../../module-reference/utility-modules/darkroom/history-stack.md) module, which lists the changes you have made, starting from the bottom -- each edit adds a new item to the top of the stack. You can select an earlier point in the history to show how the image looked at that point. The stack can be compressed to remove redundant intermediate points in your edits -- when you are happy with what you have done, just compress the history stack. Note that all edits above the selected history entry are permanently deleted when compressing the history stack.
+On the right-hand side, there is a history stack, which shows the order in which adjustments were made the the controls of the various modules. This allows you to undo changes by going back to an earlier step in the history stack.
```
Comment (wpferguson) [inline content/overview/workflow/edit-overview.md:15]: left-hand side

File: content/overview/workflow/edit-overview.md [hunk 2]
```diff
@@ -10,33 +10,39 @@ This section will guide you through the basics of developing an image in the [da
...
 
-A large number of [processing modules](../../module-reference/processing-modules/_index.md) are shipped with darktable, arranged into [module groups](../../darkroom/organization/module-groups.md). These groups are accessed via buttons at the top of the right-hand panel, just below the [histogram](../../module-reference/utility-modules/shared/histogram.md).
+# typical editing workflow
 
-# choosing a workflow
-
-When processing an image, a sequence of processing modules is applied. This sequence is known as the [pixelpipe](../../darkroom/pixelpipe/_index.md). 
+When processing an image, the order in which you make adjustments to modules is not necessarily the same as the order in which the modules appear in the pixel pipe. The following diagram shows key modulesin the pixel pipe that are typically involved in basic processing of an image, and underneath are numbers showing the order in which you would normally adjust those modules.
```
Comment (wpferguson) [inline content/overview/workflow/edit-overview.md:19]: modules in

File: content/overview/workflow/edit-overview.md (hunk 1 above)
Comment (matt-maguire) [reply content/overview/workflow/edit-overview.md]: good catch, thanks.

File: content/overview/workflow/edit-overview.md (hunk 2 above)
Comment (matt-maguire) [reply content/overview/workflow/edit-overview.md]: It's an awkward sentence; I tried to rephrase.

Comment (paperdigits) [reply content/overview/workflow/edit-overview.md]: @matt-maguire  you have a typo:

diagram shows key **modulesin** the pixel pipe

Comment (matt-maguire) [reply content/overview/workflow/edit-overview.md]: that typo was removed when I reworded the sentence.

File: content/overview/workflow/edit-overview.md [hunk 3]
```diff
@@ -6,37 +6,53 @@ weight: 20
...
 
-Two default workflows are offered in darktable, which define the default starting point for your edits:
+3. The [**_crop and rotate_**](../../../module-reference/processing-modules/crop-rotate.md) module can now be used to straighten the image and frame it correctly. You may also want to use the [_perspective correction_](../../module-reference/processing-modules/perspective-correction.md) module to apply some keystone correction so that vertical and horizontal features in the images are properly aligned. This is the digital equivalent of using a tilt-shift lens, which is commonly used when shooting images of architecture.
```
Comment (ralfbrown) [inline content/overview/workflow/edit-overview.md:37]: *crop and  rotate* is deprecated and *perspective correction* has been renamed to *rotate and perspective*

File: content/overview/workflow/edit-overview.md [hunk 4]
```diff
@@ -6,37 +6,53 @@ weight: 20
...
+5. The [**_filmic rgb_**](../../module-reference/processing-modules/filmic-rgb.md) module is now used to do a global tone mapping on the image, where we fix up the shadows and highlights in the image. First we set the white relative exposure (the auto picker next to the slider can be helpful if your image contains pure white regions). Adjust the black relative exposure if the blacks are too washed out or too crushed (it is best to do this manually, as the black auto picker can be sensitive to noise and unreliable). The _filmic rgb_ module will set the overall tone mapping for the image; to make finer adjustments to specific areas of the image, we recommend to use the [_tone equalizer_](../../module-reference/processing-modules/tone-equalizer.md) module.
```
Comment (ralfbrown) [inline content/overview/workflow/edit-overview.md:41]: Perhaps drop "and unreliable"?  The black picker is definitely sensitive to noise.

File: content/special-topics/non-recommended-modules.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+: The _monochrome_ module can be quite fiddly to use. The _color calibration_ presets better emulate what physically happens with film, or you can set the _output saturation_ slider in the _color balance_ module to 0% for a more perceptual approach.
+
+[_fill light (deprecated)_](../../module-reference/processing-modules/fill-light.md)
+: _prefer [tone equalizer](../../module-reference/processing-modules/tone-equalizer.md) (or [exposure](../../module-reference/processing-modules/exposure.md))_
+
+: Used to add light to a scene, this module again uses blurs in Lab space. The _tone equalizer_ works in linear space, or you can also achieve a similar effect by using the _exposure_ module with a [drawn mask](../../darkroom/masking-and-blending/masks/drawn.md).
+
+[_bloom_](../../module-reference/processing-modules/bloom.md)
+: _prefer [tone equalizer](../../module-reference/processing-modules/tone-equalizer.md) (or [exposure](../../module-reference/processing-modules/exposure.md))_
```
Comment (ralfbrown) [inline content/special-topics/non-recommended-modules.md:72]: *diffuse or sharpen* has a bloom preset.

**Comment by elstoc** [comment]
I think this whole PR is essentially superseded by #349 now and should be closed.

**Pull request closed without merging**